webui/react/.eslintrc.js

```diff
@@ -26,6 +26,8 @@ module.exports = {
   plugins: [ 'import', 'react', 'react-hooks', 'sort-keys-fix' ],
   root: true,
   rules: {
+    // Can disagree with @typescript-eslint/member-ordering.
+    '@typescript-eslint/adjacent-overload-signatures': 'off',
```

Comment:
I couldn't figure out how to configure the updated member-ordering to work nicely with this rule so I turn this one off for now as I don't think it's contributing much to our codebase as is. we can turn back it back on when we configure member-ordering.
https://typescript-eslint.io/rules/member-ordering/


https://typescript-eslint.io/rules/adjacent-overload-signatures/#when-not-to-use-it very helpful :))